cni/pkg/plugin/testdata/basic.txt.golden

```diff
@@ -0,0 +1,26 @@
+* nat
```

Comment:
Not in scope for this PR, but it would be nice to move the core iptables tests to this format as well; its annoying right now that golden is in code so its hard to see diffs on failure and hard to update

Reply:
Ack.